src/app/js/public/resource/Resource.js

```diff
@@ -85,7 +85,8 @@ export class ResourceComponent extends React.Component {
         if (
             match.params &&
             match.params.uri &&
-            match.params.uri.length !== 32 && // md5 length for subresources uuid
+            match.params.uri.length !== 32 &&
+            !match.params.uri.includes('%2F') && // md5 length for subresources uuid
```

Comment:
Why not allowing a html encoded slash?

Reply:
html encoded slash is the old version of sub-resources. Because Julien made the change, we didn't think that it would affect the old models. So adding this condition allows us to check both possible cases.